packages/wonder-blocks-testing/src/fixtures/fixtures.basic.stories.tsx

```diff
@@ -7,9 +7,8 @@ type Props = {
     propB?: string;
 };
 
-// @ts-expect-error [FEI-5019] - TS2322 - Type 'string' is not assignable to type 'ReactElement<any, string | JSXElementConstructor<any>>'.
 const MyComponent: React.FC<Props> = (props): React.ReactElement =>
-    `I am a component. Here are my props: ${JSON.stringify(props, null, 2)}`;
+    <>`I am a component. Here are my props: ${JSON.stringify(props, null, 2)}`</>;
```

Comment:
This kind of annoying.  Hopefully they improve the React types to allow returning strings from functional components.

Reply:
I'm not sure if I did this right, since I think I may need to wrap the backticks in `{}`.  I double check this.

Reply:
I got rid of this and added a suppression comment in a later PR in the stack.